<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DualListBox</class>
 <widget class="QMainWindow" name="DualListBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DualListBox</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonPlot">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>535</y>
      <width>70</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Plot</string>
    </property>
    <property name="shortcut">
     <string>Alt+P</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSelectAllRight">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>415</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select &amp;All</string>
    </property>
    <property name="shortcut">
     <string>Alt+A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAdd">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>535</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Run</string>
    </property>
    <property name="shortcut">
     <string>Alt+R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDone">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>535</y>
      <width>70</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Done</string>
    </property>
    <property name="shortcut">
     <string>Alt+D</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarRunMC">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>475</y>
      <width>590</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeselectAllRight">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>415</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Deselect Al&amp;l</string>
    </property>
    <property name="shortcut">
     <string>Alt+L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRun">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>535</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Clean and Run</string>
    </property>
    <property name="shortcut">
     <string>Alt+C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSelectAllLeft">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>415</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Select All</string>
    </property>
    <property name="shortcut">
     <string>Alt+S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeselectAllLeft">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>415</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Deselect All</string>
    </property>
    <property name="shortcut">
     <string>Alt+D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonToLeft">
    <property name="geometry">
     <rect>
      <x>306</x>
      <y>345</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCancel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>535</y>
      <width>70</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Cancel</string>
    </property>
    <property name="shortcut">
     <string>Alt+C</string>
    </property>
   </widget>
   <widget class="QLabel" name="textLabelMachine">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>65</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Machine Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonToRight">
    <property name="geometry">
     <rect>
      <x>306</x>
      <y>285</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>--&gt;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxOffAxis">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>535</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>OffAxis</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="textLabelTitle">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>320</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;p align=&quot;center&quot;&gt;&lt;b&gt;&lt;font size=&quot;+2&quot;&gt;Verification of Beam Model&lt;/font&gt;&lt;/b&gt;&lt;/p&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="listBoxLeft">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>256</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listBoxRight">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>256</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
